### PR TITLE
Use `DOMParser` and `textContent` to extract image author

### PR DESCRIPTION
**Pull request by zstadler**

### CODE REVIEW

File: IsraelHiking.Web/src/application/services/image-attribution.service.ts [hunk 1]
```diff
@@ -16,6 +16,12 @@ export class ImageAttributionService {
     private readonly resources = inject(ResourcesService);
     private readonly httpClient = inject(HttpClient);
 
+    private extractPlainText(html: string): string {
+        const parser = new DOMParser();
+        const doc = parser.parseFromString(html, 'text/html');
+        return doc.documentElement.textContent.replace(/( *\n *)+/g, '\n').replace(/ +/g, ' ').trim();
```
Comment (HarelM) [inline IsraelHiking.Web/src/application/services/image-attribution.service.ts:22]: Should this be `\s` instead of " " (regular space)?

**Comment by HarelM** [comment]
This looks like a good solution to replace the regular expression I wrote back then.
Thanks!

**Comment by HarelM** [comment]
I've moved it to another brach with more fixes. Thanks!

File: IsraelHiking.Web/src/application/services/image-attribution.service.ts (hunk 1 above)
Comment (zstadler) [reply IsraelHiking.Web/src/application/services/image-attribution.service.ts:22]: `\s` includes `\n` and I wanted to preserve them using `replace(/( *\n *)+/g, '\n')`

**Pull request closed without merging**